gremlin-go/driver/client.go

```diff
@@ -42,8 +42,8 @@ type ClientSettings struct {
 	NewConnectionThreshold int
 	// Maximum number of concurrent connections. Default: number of runtime processors
 	MaximumConcurrentConnections int
-	
-	Session         string
+
+	Session string
```

Comment:
nit: this format seems a bit off. Did you run `go fmt`?

Reply:
`go fmt` don't remove empty lines, fixed manually https://github.com/lyndonbauto/tinkerpop/pull/68/commits/d85a4d6b05d4f3a509b3e681af1dab9f86741a5d